ipfs-cluster-ctl/main.go

```diff
@@ -126,11 +127,38 @@ requires authorization. implies --https, which you can disable with --force-http
 			Name:  "force-http, f",
 			Usage: "force HTTP. only valid when using BasicAuth",
 		},
+		cli.StringFlag{
+			Name:  "trace-out",
+			Value: "",
+			Usage: "output path for tracing result. If not provided, results will be written to std.",
+		},
 	}
 
+	rootCtx := context.Background()
+
 	app.Before = func(c *cli.Context) error {
 		cfg := &client.Config{}
 
+		rootSpanName := "ctl"
```

Comment:
Probably put all tracing setup in its own function, but please disable by default.